mystery-deckbuilder/Assets/Scripts/Card/Cards.cs

```diff
@@ -0,0 +1,259 @@
+
+
+
+
+
+/*
+ * Static class containing a method to return an instance of a card with a given ID
+ * Returned object must be cast to it's card type (Conversation/Preparation)
+ */
+public static class Cards
+    {
+    public static object CreateCardWithID(int id)
+    {
+        switch (id)
+        {
+            case 1:
+                return new Bluster();
+            case 2:
+                return new PartingShot();
+            case 3:
+                return new BrowBeat();
+            case 4:
+                return new BadCop();
+            case 5:
+                return new Encourage();
+            case 6:
+                return new Complement();
+            case 7:
+                return new SobStory();
+            case 8:
+                return new GoodCop();
+            case 9:
+                return new Articulate();
+            case 10:
+                return new Salutation();
+            case 11:
+                return new Lecture();
+            case 12:
+                return new MenacingPresence();
+            case 13:
+                return new Tirade();
+            case 14:
+                return new Empathize();
+            case 15:
+                return new Reassure();
+            case 16:
+                return new Eloquence();
+            case 17:
+                return new Monologue();
+            default:
+                return null;
+        }
+    }
```

Comment:
NIT: This pattern is okay, but you now need to make sure that these stay updated with new cards!